metricbeat/module/system/filesystem/helper_test.go

```diff
@@ -43,6 +43,10 @@ func TestFileSystemList(t *testing.T) {
 	assert.True(t, (len(fss) > 0))
 
 	for _, fs := range fss {
+		if (fs.TypeName == "removable" || fs.TypeName == "") && runtime.GOOS == "windows" {
+			continue
+		}
```

Comment:
If this is the solution this may be hiding some real bug, do CI machines have removable devices? Or is it possible to have a device without type?

Reply:
I'm not sure if you have read the description above but, in any case, maybe the question is: do you have a better idea? :)

Reply:
These should be the only possible types:
```
                DRIVE_UNKNOWN:     "unknown",
		DRIVE_NO_ROOT_DIR: "invalid",
		DRIVE_REMOVABLE:   "removable",
		DRIVE_FIXED:       "fixed",
		DRIVE_REMOTE:      "remote",
		DRIVE_CDROM:       "cdrom",
		DRIVE_RAMDISK:     "ramdisk",
```
and `removable` should also work in this case. I would first start with enriching the error message with more information on the E, F, G drives, maybe their actual drive type (I could not see it in the error message). Let me know if you need any help testing on windows here.